### PR TITLE
Serialization example, plus fixes to paths

### PR DESCRIPTION
**Pull request by geoalgo**
… to serialize model

*Issue #126 

*Description of changes:*

Add an example to show how models can be serialized, also set a path to download datasets and models that is independent of where scripts are run (similar to what gluoncv or gluonnlp do).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-143/1 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-143/1/index.html

File: src/gluonts/support/util.py [hunk 1]
```diff
@@ -470,3 +471,18 @@ def erf(F, x: Tensor):
...
     return F.where(F.broadcast_greater_equal(x, zeros), res, -1.0 * res)
+
+
+def get_download_path() -> Path:
+    """
+
+    Returns
+    -------
+    Path
+        default path to download datasets, models of gluon-ts, for instance /home/username/.mxnet/gluon-ts/
```
Comment (aalexandrov) [inline src/gluonts/support/util.py]: Minor comment, but I think this line exceeds the 79 chars limit (black does not enforce auto line-wrap at the moment).

File: examples/persist_model.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    estimator = SimpleFeedForwardEstimator(
+        prediction_length=dataset.metadata.prediction_length,
+        freq=dataset.metadata.time_granularity,
+        trainer=Trainer(epochs=5, num_batches_per_epoch=10),
+    )
+
+    predictor = estimator.train(dataset.train)
+
+    # save the trained model in a path ~/username/.mxnet/gluon-ts/feedforward/
```
Comment (aalexandrov) [inline examples/persist_model.py]: in a path `~/.mxnet/gluon-ts/feedforward/` (`~` expands to the user home already)

**Comment by szha** [comment]
Job PR-143/2 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-143/2/index.html

File: examples/persist_model.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    model_path = get_download_path() / "feedforward"
+    os.makedirs(model_path, exist_ok=True)
+
+    predictor.serialize(model_path)
+
+    # loads it back and evaluate predictions accuracy with the deserialized model
+    predictor_deserialized = Predictor.deserialize(model_path)
+
+    forecast_it, ts_it = make_evaluation_predictions(
+        dataset.test, predictor=predictor, num_eval_samples=100
```
Comment (benidis) [inline examples/persist_model.py]: Shouldn't this be `predictor=predictor_deserialized`?

**Pull request merged**